tests/Pool.R

```diff
@@ -2,11 +2,11 @@ library(DBI)
 library(pool)
   pool <- dbPool(
     drv = RIBMDB::ODBC(),
-    dbname = "bjhadb",
+    dbname = "SAMPLE"
     host = "waldevdbclnxtst06.dev.rocketsoftware.com",
     port = 60000,
-    user = "newton",
-    password = "A2m8test"
+    user = "db2user",
+    password = "xxxxxx"
```

Comment:
@vmathur12 Update code to read userid and passwd from ENV variable as we did for go driver. Thanks.